host.json

```diff
@@ -0,0 +1,27 @@
+{
+        "name": "contivModel",
+        "objects": [
+                {
+                        "name": "Bgp",
+                        "type": "object",
+                        "key": ["name"],
+      "properties": {
```

Comment:
Itseems like Indentation is little of here. You might want to check your editor settings.